### PR TITLE
Working splash page

### PR DESCRIPTION
**Pull request by dhunninghake**

### CODE REVIEW

**Comment by dhunninghake** [comment]
We had issues getting Webhook to deliver data from the CMS, so for now it's a static build.

File: libs/.firebase.conf.jst [hunk 1]
```diff
@@ -1,6 +1,4 @@
 {
-  "secretKey"   : "<%= secretKey %>",
-  "siteName"    : "<%= siteName %>"<% if (noSearch) { %>,
-  "noSearch"    : <%= noSearch %><% } %><% if (imageproxy) { %>,
-  "imageproxy"  : "<%= imageproxy %>"<% } %>
+  "secretKey"   : "9f4f343c-1c2d-48f2-9ca2-818b83c4a391",
+  "siteName"    : "sanctucompu"
 }
```
Comment (keithpops) [inline libs/.firebase.conf.jst]: Might be worth adding this to the `.gitignore` file so we don't commit any keys

**Review by keithpops — CHANGES_REQUESTED** [review]
Let's also add `main.js` to the the `.gitignore` file since that will be recompiled by webhook anyways

**Comment by hhff** [comment]
👍 agree w/ keith- happy to merge after those comments!

**Review by keithpops — APPROVED** [review]

**Pull request merged**